package.json

```
@@ -69,6 +70,12 @@
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
   },
+  "standard-version": {
+    "scripts": {
+      "postbump": "ts-node ./scripts/upgradeConfig.ts",
```

```suggestion
      "postbump": "npm run upgrade-config",
```
Should prevent having duplicate scripts in general.

---

Actually might even just remove the script, originally added it to use the npm env variables. It also won't be executed separately (and if needed one can still do so using ts-node)